media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
So to try and re-iterate what I think needs doing here:

1.) fix the overall width of `#language-search` at tablet break-point so it doesn't overflow it's parent container width (do this by changing the width of the child elements of `#language-search`, instead of trying to override the width here.
2.) Try setting .`search-column` to a slightly more user friendly, not so small width at tablet break-point.